assets/less/formstyles.less

```diff
@@ -345,13 +353,12 @@ input.errorHighlight {
       background-color: @hoverHighlight;
       color: #fff;
       .box-shadow(none);
-
     }
     &.active {
+      z-index: 1 !important;
```

Comment:
Maybe it would be relevant to add a comment here to justify the !important

Reply:
you got it